### PR TITLE
Validate JSON on pinned data save

### PR DESCRIPTION
**Pull request by ivov**

### CODE REVIEW

File: packages/editor-ui/src/plugins/i18n/locales/en.json [hunk 1]
```diff
@@ -624,12 +624,13 @@
 	"runData.copyParameterPath": "Copy Parameter Path",
 	"runData.copyToClipboard": "Copy to Clipboard",
 	"runData.copyValue": "Copy Value",
-	"runData.editValue": "Edit Value",
+	"runData.editOutput": "Edit Output",
```
Comment (ivov) [inline packages/editor-ui/src/plugins/i18n/locales/en.json:627]: From handoff:

> Hovering on Edit (pencil) action should have alt label of `Edit output`

File: packages/editor-ui/src/components/RunData.vue [hunk 2]
```diff
@@ -533,6 +533,19 @@ export default mixins(
...
+				} catch (error) {
+					const title = this.$locale.baseText('runData.invalidPinnedData');
+
+					const toRemove = new RegExp(/JSON\.parse:|of the JSON data/, 'g');
+					const message = error.message.replace(toRemove, '').trim();
+					error.message = message.charAt(0).toUpperCase() + message.slice(1);
+
+					this.$showError(error, title);
+					return;
+				}
```
Comment (alexgrozav) [inline packages/editor-ui/src/components/RunData.vue:547]: Should we also check for a specific structure? Is the value supposed to be an Array of objects?

Comment (ivov) [reply packages/editor-ui/src/components/RunData.vue:547]: We will be using `pinData` mirroring how `runData` works. I would wait to debug the DS once we connect and can test out how pin data flows.

Comment (alexgrozav) [reply packages/editor-ui/src/components/RunData.vue:547]: Got it! I'll merge this meanwhile.

**Pull request merged**